package.json

```diff
@@ -4,7 +4,8 @@
   "description": "API to process HTTP requests to access information on local cats",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node ./bin/www",
+    "test": "mocha 'test/apiTest.js'" 
```

Comment:
This fails for me with the following error:

>sh: mocha: command not found

Setting up CI in a similar way to the previous project would help avoid breaking the build on other developer's machines - perhaps you could try a quick PR to the master branch tomorrow?